src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -412,4 +416,15 @@ public String inviteUser(String role) {
         activeUserId = (String) pumInternalUserResponse.get("userIdentifier");
         return bearerToken;
     }
+
+    public void findMFAByUserIDScenario() {
+        findMFAByUserIDShouldBeSuccess();
+    }
+
+    public void findMFAByUserIDShouldBeSuccess() {
```

Comment:
Any reason for creating another method instead of using the above created one?

Reply:
The reason behind the first method was to make it more of a generic method, just in case in the future we would like to retrieve the mfa using some other parameters and not just by user id, so they can all be grouped under the same main scenario. I have changed it to findMFAScenario() instead of findMFAByUserIDScenario() to make it generic. But we can just make it one if it's unnecessary.

Reply:
No issues, we can leave as such. Thanks for the explanation :)

Reply:
sonar won't complain on method 3 capital letters in sequence.